docs/40_smart-contract-guides/index.md

```diff
@@ -25,3 +25,8 @@ Work your way through the following topics to build and deploy smart contracts.
 * [Inline Actions to external contracts](70_inline-action-to-external-contract.md): How to call actions in other smart contracts
 * [Linking custom permissions](80_linking-custom-permission.md): How to set up custom permissions on a smart contract
 * [Payable Actions](90_payable-actions.md): How to be paid for calling smart contract actions
+
+### EOSIO Training and Certification Center
```

Comment:
You can just remove "following" from the first sentence without changing the meaning.

Consider converting/presenting this section as an information callout for more visibility.